x11-wm/jwm/metadata.xml

```diff
@@ -19,4 +19,39 @@
 	such as Puppy Linux and Damn Small Linux, and it is available as a separate
 	package in many other distributions.
 	</longdescription>
+	<use>
+		<flag name="cairo">
+			SVG icons and backgrounds.
+		</flag>
+		<flag name="debug">
+			Debug build.
+		</flag>
+		<flag name="iconv">
+			Character set conversion.
+		</flag>
+		<flag name="jpeg">
+			JPEG icons and backgrounds.
+		</flag>
+		<flag name="nls">
```

Comment:
This is already a global USE flag (it's in profiles/use.desc) so you don't need to override it. You can override global USE flags if you feel your description adds a lot on top of the global one though.

This applies to some of the others too.

Reply:
Is this better or should I remove bidi/pango too?